Array/src/ArrayTask4.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class ArrayTask4 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String userAction;
+        String enterWords;
+        String[] words;
+        String enterValue;
+        int enterInt;
```

Comment:
Same as above, please change to one line declaration

Reply:
Done